api_v3/lib/KalturaRequestDeserializer.php

```diff
@@ -416,6 +412,15 @@ private function buildObject(KalturaTypeReflector $typeReflector, &$params, $obj
 		return $obj;
 	}
 	
+	protected function generateMultiLingualStringFromString($object, $fieldName, $value)
```

Comment:
you are not passing by reference so its not clear to me how the updates you do work